plugin/src/main/java/gorm/tools/compiler/GormRepository.java

```diff
@@ -0,0 +1,14 @@
+package gorm.tools.compiler;
+
+import org.codehaus.groovy.transform.GroovyASTTransformationClass;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@GroovyASTTransformationClass("gorm.tools.compiler.GormRepositoryArtefactTypeTransformation")
+public @interface GormRepository {
```

Comment:
move to groovy

Reply:
Tried, but was unaccessible from Java AST class.

Reply:
I don't see any other plugins that have these in the java dir.  should not be neccessary. the annotation in the gorm-rest-api isn't in the java dir. not clear are all why this is needed